org.osgi.test.cases.typedevent/src/org/osgi/test/cases/typedevent/junit/TestEvent.java

```diff
@@ -0,0 +1,16 @@
+package org.osgi.test.cases.typedevent.junit;
+
+import org.osgi.dto.DTO;
+
+public class TestEvent extends DTO {
+
+	public TestEvent(String test, String a, String A) {
```

Comment:
DTOs have no behavior which means no constructors. While the spec supports DTO-like object, this class extends DTO but violates the DTO spec.